modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaConstructorParametersCheck.java

```diff
@@ -303,8 +303,8 @@ private String _fixPassedInVariables(
 	}
 
 	private String _fixPassedInVariables(
-		String absolutePath, String content, List<JavaParameter> parameters,
-		String fileContent) {
+		String absolutePath, String content, String fileName,
+		List<JavaParameter> parameters, String fileContent) {
```

Comment:
@TamyrisTorres, is it very costly to sort all parameters alphabetically?

If not, please do it to mitigate the risk of Brian complain about it.